README.md

```diff
@@ -25,10 +25,11 @@ As a successor of [v2rayA](https://github.com/v2rayA/v2rayA), dae abandoned v2ra
 - [x] Support to automatically switch nodes according to policy. That is to say, support to automatically test independent TCP/UDP/IPv4/IPv6 latencies, and then use the best nodes for corresponding traffic according to user-defined policy.
 - [x] Support advanced DNS resolution process.
 - [x] Support full-cone NAT for shadowsocks, trojan(-go) and socks5 (no test).
+- [x] Support `tuic` protocol.
```

Comment:
Why not add other protocols here?

Reply:
Let's put a ref link to https://github.com/daeuniverse/dae/blob/docs_refactor/docs/en/other-proxy-protocol.md

Also, I recommend renaming to `proxy-protocols.md` instead of `other-proxy-protocol.md`

Reply:
Resolved in [refactor: rename other-proxy-protocol.md -> proxy-protocols.md](https://github.com/daeuniverse/dae/pull/179/commits/ff304a7a546d9ee93740d7157c7cc76e144d746e) and [docs(readme): add ref to proxy-protocols.md](https://github.com/daeuniverse/dae/pull/179/commits/a154b94d68a43af9024799a7c0a226f723b62680)